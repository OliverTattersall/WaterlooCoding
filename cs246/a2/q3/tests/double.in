a 0 b 1 1 4 4
a 0 b 1 1 4 4
pc0
r 0 0
pc0
