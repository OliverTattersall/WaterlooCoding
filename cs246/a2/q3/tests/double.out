Dimensions: 5x5
Number of rectangles: 2
	[Blue (1,1) 4x4]
	[Blue (1,1) 4x4]
Dimensions: 5x5
Number of rectangles: 1
	[Blue (1,1) 4x4]
